core/spring-authorization-server-core.gradle

```diff
@@ -18,3 +19,7 @@ dependencies {
 
 	provided 'javax.servlet:javax.servlet-api'
 }
+
+jacoco {
```

Comment:
Is it necessary to include this lib?

Reply:
Yes. This is the same issue I faced in my PR gh-70. Build fails with JDK 14 if jacoco `0.8.5` is not used.